### ISSUE TITLE
matogro: support chapters or sections

### ISSUE THREAD

**Issue by sschober** [sschober/mato#24]

**Comment by sschober** [comment]
I think there are no "sections" as such in mom - only chapters, so I will consider using these.

It seems to me, `.CHAPTER <n>` macros have to be declared outside `.START` and `.COLLATE` pairs.
Currently, `.START` is the last macro of the default preamble. This hints at a new mechanism for metadata - see #20

**Comment by sschober** [comment]
I think, it would be a good idea, to create a metadata header `doctype: chapter`, which might alter the behaviour of headings. Every heading level is shift to one lower and the top level is in fact a chapter:

```
---
doctype: chaper
---
# Chapter 1 Title of chapter
## Heading 1
...
```

**Comment by sschober** [comment]
And, BTW, `.CHAPTER` does not work in slides at all.

There, I should resolve to level 1 headings, which are rendered v-centered and bigger, or something and use level 2 headers on slides.

**Comment by sschober** [comment]
this has now been implemented fully via #20
